tests/test_config.py

```diff
@@ -297,6 +303,9 @@ def test_parse_config(tmp_path: Path) -> None:
     assert one.image_version == "1.2.3"
     assert two.image_version == "4.5.6"
 
+    # Default values from subimages should be picked up.
+    assert len(two.package_manager_trees) == 1 and two.package_manager_trees[0].source.name == "mkosi.skeleton"
```

Comment:
This should probably also assert something about `one`
```suggestion
    assert len(one.package_manager_trees == 0 and len(two.package_manager_trees) == 1
    assert two.package_manager_trees[0].source.name == "mkosi.skeleton"
```